empire/server/modules/powershell/persistence/misc/add_sid_history.yaml

```diff
@@ -26,7 +26,11 @@ options:
     required: true
     value: ''
   - name: Groups
```

Comment:
Thanks for that. That makes sense to me then, just update the yaml then so its 'Group' so the variable is called correct in the py file.

Reply:
Got it, my bad. Thanks